DirectProgramming/DPC++FPGA/Tutorials/Features/kernel_args_restrict/README.md

```diff
@@ -8,14 +8,13 @@ The [oneAPI Programming Guide](https://software.intel.com/en-us/oneapi-programmi
 
 | Optimized for                     | Description
 ---                                 |---
-| OS                                | Linux* Ubuntu* 18.04; Windows* 10
-| Hardware                          | Intel® Programmable Acceleration Card (PAC) with Intel Arria® 10 GX FPGA; <br> Intel® FPGA Programmable Acceleration Card (PAC) D5005 (with Intel Stratix® 10 SX)
+| OS                                | Linux Ubuntu 18.04/20.04; RHEL/CentOS 8; SUSE 15; Windows 10
```

Comment:
Did you intend to delete the '*' from all the OS names, or was that an oversight?  I believe the '*' is still used elsewhere in the file, so this is at minimum inconsistent.

Reply:
Heh, I assume those italics are a reference to the missing * s.
It was looking awfully busy so I removed them, but that was a mistake. Fixed in the latest commit.